src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -2117,8 +2117,8 @@ private BoundExpression BindCastCore(ExpressionSyntax node, BoundExpression oper
                 case BoundKind.TupleLiteral:
                     {
                         var tuple = (BoundTupleLiteral)operand;
-                        var targetElementTypes = default(ImmutableArray<TypeSymbolWithAnnotations>);
-
+                        _ = default(ImmutableArray<TypeSymbolWithAnnotations>);
```

Comment:
`_ = constant` seems like it should just be removed.

Reply:
Havin this feature go in and tell users to go split their single line into multiple lines seems like it's actually a cure worse than the disease.  For these cases, where there is no side-effects in the initializer, we should just remove entirely.  

note: i actually think this is important, and we should strongly consider tweaking that behavior before general release.